src/schema/v2/artwork/__tests__/artwork.test.js

```diff
@@ -4269,4 +4269,24 @@ describe("Artwork type", () => {
       })
     })
   })
+
+  describe("#priceListed", () => {
+    const query = `
+      {
+        artwork(id: "richard-prince-untitled-portrait") {
+          priceListed
+        }
+      }
+    `
+
+    it("returns artworks price_listed", () => {
+      artwork.price_listed = 123
+      artwork.price_currency = "USD"
```

Comment:
It doesn't hurt the test, but is this line necessary?

Reply:
Nope, this was when I had it as `Money`, I will remove it.